jobs/ci-kubernetes-e2e-gce-gpu-1-7.env

```diff
@@ -0,0 +1 @@
+PROJECT=k8s-jkns-e2e-gce-gpus-1-7
```

Comment:
Submitted a quota request. This PR can go in. Once the request is approved I expect the tests to go green